batch-setup/make_meta_tiles.py

```diff
@@ -195,14 +197,18 @@ def enqueue_tiles(config_file, tile_list_file):
                 print("All done!")
                 break
 
+            check_metatile_exists = retry_number > 0
```

Comment:
It might be good to add a comment here about why you override the metatile existence check.

Reply:
6a72420